.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,47 @@
+name: Bug Report
+description: Report a bug or unexpected behaviour with the website.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this bug!
+
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: Reproduction steps
+      description: "Please describe how to reproduce the bug."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-results
+    attributes:
+      label: Expected results
+      description: "What should happen when you follow the reproduction steps?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-results
+    attributes:
+      label: Actual results
+      description: "What actually happened?"
+    validations:
+      required: true
+
+  - type: input
+    id: device-and-os
+    attributes:
+        label: What device are you using and what OS is it running?
```

```suggestion
        label: Device and OS
        description: The type of device and operating system you were using when the error occurred.
```
Missed this in my previous pass, sorry.